CHANGELOG.md

```diff
@@ -82,6 +82,8 @@
   [#10346](https://github.com/Kong/kong/pull/10346)
 - Fix an issue where control plane does not rename fields correctly for `session` for older version of data planes.
   [#10352](https://github.com/Kong/kong/pull/10352)
+- Fix an issue where validation to regex routes may be skipped when the old-fashioned config is used for DB-less Kong.
```

Comment:
```suggestion
- Fix an issue where migration to regex routes may be skipped when the old-fashioned config is used for DB-less Kong.
```

Reply:
No. It's not the migration being skipped. The migration will happen anyway.

Reply:
So the issue is about validation being skipped? Aka it should fail validation, but now it does not fail?